src/app/views/marketplace/marketplace-app.html

```diff
@@ -7,17 +7,22 @@
         <i class="fa fa-chevron-left"></i> {{ 'mno_enterprise.templates.dashboard.marketplace.show.back' | translate }}
       </a>
       <div class="media" ng-show="!vm.isLoading">
-        <div class="app-rating pull-right">
-          <div ng-show="vm.isRateDisplayed">
-            <average-star-rating ng-model="vm.averageRating" display-rating="true" max="5"></average-star-rating>
-          </div>
-        </div>
         <div class="pull-left">
           <img class="media-object" ng-src="{{vm.app.logo}}" ng-alt="{{vm.app.name}}">
         </div>
         <div class="media-body">
           <h2 class="media-heading">{{vm.app.name}}</h2>
         </div>
+        <div class="app-rating pull-left">
+          <div ng-show="vm.isRateDisplayed">
+            <average-star-rating ng-model="vm.averageRating" display-rating="true" max="5"></average-star-rating>
+          </div>
+        </div>
+        <div class="pull-left links">
+          <a ng-show="vm.anyReviews" ng-click="vm.scrollToReviews()" href="">{{vm.reviews.list.length}} {{'mno_enterprise.templates.dashboard.marketplace.show.reviews' | translate}}</a>
+          <span ng-if="vm.anyReviews && vm.anyQuestions">  |  </span>
```

Comment:
Use `&nbsp;`: `&nbsp;|&nbsp;`
I'm afraid that the code uglifier will remove those spaces after a build.